lib/fibonacci.rb

```diff
@@ -5,8 +5,18 @@
 # ....
 # e.g. 6th fibonacci number is 8
 
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: 0(n), where the program only loops through it n times (well, n - 2 times because the first few things are hardcoded cases), was wondering if it was better to do it with recursion? but that would run it with recursion instead of saving everything in an array and indexing it, but I think that would run more times?
```

Comment:
I kind of get what `n` is here, but `loops through it n times` is a bit of a tautology.

Can you explain it more clearly perhaps? e.g. `O(n), where n = the number of digits in the input number` or `O(n), where n = the input number itself.`